SDK.md

```diff
@@ -128,7 +128,7 @@ Here's the list of getters:
     Source() -> URL
     EventID() -> string
     EventTime() -> RFC3339
-    SchemaURL() -> string
+    DataSchema() -> string
```

Comment:
URL

Reply:
Makes sense to me - I'll pick up this change - if someone thinks it's incorrect we can open another PR and revert it.  @clemensv @markpeek @fabiojose @deissnerk